styles.css

```diff
@@ -428,45 +428,38 @@ nav {
   font-family: Helvetica, Arial, sans-serif;
 }
 
-.animation  {
+.animation {
   position: relative;
   top: 50%;
   left: 0px;
-  animation: fly .5s linear forwards;
+  animation: fly 0.5s linear forwards;
   animation-timing-function: ease-out;
-  animation-name: flying; 
+  animation-name: flying;
   position: relative;
   top: 50%;
   left: 0px;
-  animation: fly .5s linear forwards;
-  
-   }
-  
+  animation: fly 0.5s linear forwards;
+}
 
- @keyframes fly {
+@keyframes fly {
   0% {
-   transform: translateX(0px);
+    transform: translateX(0px);
   }
   90% {
-   transform: translateX(1000px);
+    transform: translateX(1000px);
   }
-   100% {
+  100% {
     opacity: 0;
     display: none;
-   }
   }
- 
+}
```

Comment:
please remove this - not needed

Reply:
I don't have it in my last version of css